src/VideoToolbox/VTVideoEncoder.cs

```diff
@@ -166,52 +173,54 @@ internal VTVideoEncoder (NSDictionary dict)
 			// added in Xcode 11 so the constants won't exists in earlier SDK, making all values optional
 
 			var constant = VTVideoEncoderList.GpuRegistryId;
-			if (constant != null) {
+			if (constant is not null) {
 				var gri = dict [constant] as NSNumber;
 				GpuRegistryId = gri?.UInt64Value; // optional
 			}
 
 			constant = VTVideoEncoderList.SupportedSelectionProperties;
-			if (constant != null) {
+			if (constant is not null) {
 				if (dict.TryGetValue (constant, out NSDictionary d)) // optional
 					SupportedSelectionProperties = d;
 			}
 
 			constant = VTVideoEncoderList.PerformanceRating;
-			if (constant != null) {
-				PerformanceRating = dict [constant] as NSNumber; // optional
+			if (constant is not null) {
+				if (dict [constant] is NSNumber performanceRatingNum)
```

Comment:
Merge them in and &&

Reply:
`PerformanceRating` is nullable (`NSNumber?`), so why is this change needed?